src/UI/Factory.php

```diff
@@ -253,4 +253,22 @@ public function listing();
 	 * @return \ILIAS\UI\Component\Image\Factory
 	 */
 	public function image();
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Panels are used to group titled content.
+	 *   composition: >
+	 *      Panels consist of a header and content section. They form one Gestalt and so build a perceivable
+	 *      cluster of information.
+	 *   effect: The effect of interaction with panels heavily depends on their content.
+	 *
+	 * rules:
+	 *   wording:
+	 *      1: Panels MUST contain a heading.
```

Comment:
Maybe change to "Panels MUST have a title." as this is the parameter we actually supply to the panel constructors?